carbon-design-system/carbon-addons-iot-react
# feat(table): update batch cancel button label

Closes #3737

**Summary**

- Change batch cancel button label from "Cancel" to "Clear selections"

**Change List (commits, features, bugs, etc)**

- Update `i18n` object for Table
- Update tests
- Update cypress image snapshots

**Acceptance Test (how to verify the PR)**

- Go to [this story](https://deploy-preview-3780--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table--with-selection-and-batch-actions)
- Verify that batch action button label is "Clear selections"

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
